spec/controllers/posts_controllers_spec.rb

```diff
@@ -0,0 +1,35 @@
+require 'rails_helper'
+
+RSpec.describe PostsController, type: :controller do
```

Comment:
- [ ] These tests failed and I have attached an image of the failing tests. Please use `type: :request` instead `type: :controller` as it a requirement for this project. See example shown below
 ```ruby
RSpec.describe "Posts", type: :request do
   ...
 end
```

<img width="530" alt="slack_zTgAIJbjC4" src="https://user-images.githubusercontent.com/81468675/178562408-4ca1b785-2d8c-419a-8c1f-ffa7034120cc.png">


- [ ] Please take a look at this second image and fix all the files that are modified, the views should have `posts/index.html.erb` and `posts/show.html.erb` and not `posts/user.html.erb`. Kindly fix this in the views of the users too.

- [ ] See the example of how to write the request test in the image and please use it as an example to modify all your tests so that they all can pass.


<img width="644" alt="Code_tnWwYP5ACb" src="https://user-images.githubusercontent.com/81468675/178567830-e61be707-0fda-4131-aa32-60441f4f4e07.png">